templates/fast-api/pyproject.toml

```diff
@@ -10,6 +10,7 @@ version = "0.1.0"
 common-grants-sdk = {path = "../../lib/python-sdk", develop = true}
 fastapi = {extras = ["standard"], version = "^0.115.11"}
 python = "^3.11"
+pyyaml = "^6.0.2"
```

Comment:
I think it's good to have this dependency because YAML is easier to read, but I'm also updating the `check spec` command to accept OpenAPI specs in JSON in case that was an issue you bumped into.